promotion/promotion.go

```diff
@@ -51,13 +52,19 @@ func (service *Service) GetAvailablePromotions(
 	migrate bool,
 ) (*[]Promotion, error) {
 	if walletID != nil {
+		l := zerolog.Ctx(ctx)
+		l.UpdateContext(func(c zerolog.Context) zerolog.Context {
+			return c.Str("walletID", (*walletID).String())
```

Comment:
one small ask, can you quickly update this to use our helper method? otherwise looks great, thank you @maikelmclauflin